src/components/layout/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import Footer from './Footer'
-import Header from './Header'
+import Footer from '~Components/layout/footer'
+import Header from '~Components/layout/header'
```

Comment:
It's better to import in the following way:
```suggestion
import Footer from './Footer'
import Header from './Header'
```
Because this component is at the same level as the rest of them. The objective of the _aliases_ is to have shorter imports, so this way will be worst in this case. Additionally, "_Header_" and "_Footer_" are internal components of "_Layout_", ergo you will only use them here. You have not to export them for the rest of our application.

Reply:
Oh! I understand. Thanks you for the explanation, totally agree.